DataLakeModels/Models/Reels/Reel.cs

```diff
@@ -0,0 +1,110 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace DataLakeModels.Models.Reels {
+    public class Reel : IEquatable<Reel> {
+        public string Id { get; set; }
+        public string Pk { get; set; }
+        public string Code { get; set; }
+        public User User { get; set; }
+        public string UserId { get; set; }
+        public Caption Caption { get; set; }
+        public ICollection<CommentInfo> Comments { get; set; }
+        public long TakenAt { get; set; }
+        public bool HasAudio { get; set; }
+        public bool HasLiked { get; set; }
+        public long MediaType { get; set; }
+        public long FilterType { get; set; }
+        public bool PhotoOfYou { get; set; }
+        public string ProductType { get; set; }
+        public ClipsMeta ClipsMetaData { get; set; }
+        public long DeletedReason { get; set; }
+        public string MusicMetadata { get; set; }
+        public long OriginalWidth { get; set; }
+        public double VideoDuration { get; set; }
+        public ICollection<VideoVersion> VideoVersions { get; set; }
+        public bool CanViewerSave { get; set; }
+        public ImageVersion ImageVersions { get; set; }
+        public long OriginalHeight { get; set; }
+        public string ClientCacheKey { get; set; }
+        public long DeviceTimestamp { get; set; }
+        public DateTime CreateTime { get; set; }
+        public bool IsUnifiedVideo { get; set; }
+        public ICollection<CommentInfo> PreviewComments { get; set; }
+        public bool CaptionIsEdited { get; set; }
+        public bool HasMoreComments { get; set; }
+        public bool IsInProfileGrid { get; set; }
+        public string LoggingInfoToken { get; set; }
+        public bool IsPaidPartnership { get; set; }
+        public SquareCrop MediaCroppingInfo { get; set; }
+        public string CommercialityStatus { get; set; }
+        public bool CommentLikesEnabled { get; set; }
+        public Friction SharingFrictionInfo { get; set; }
+        public string OrganicTrackingToken { get; set; }
+        public string CommentInformTreatment { get; set; }
+        public bool ShouldHaveInformTreatment { get; set; }
+        public bool CanSeeInsightsAsBrand { get; set; }
+        public bool CommentThreadingEnabled { get; set; }
+        public string IntegrityReviewDecision { get; set; }
+        public bool ProfileGridControlEnabled { get; set; }
+        public bool LikeAndViewCountsDisabled { get; set; }
+        public bool CanViewMorePreviewComments { get; set; }
+        public bool HideViewAllCommentEntrypoint { get; set; }
+        public long MaxNumVisiblePreviewComments { get; set; }
+        public bool OriginalMediaHasVisualReplyMedia { get; set; }
+        public bool IsVisualReplyCommenterNoticeEnabled { get; set; }
+
+        public ICollection<ReelStats> Stats { get; set; }
+
+        bool IEquatable<Reel>.Equals(Reel other) {
```

Comment:
I think this is too much information. Not only it makes hard to debug, but might hurt performance, if we need to check new entries against thousands of existing entries